stable/acs-engine-autoscaler/templates/deployment.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if and .Values.acsenginecluster.resourcegroup .Values.acsenginecluster.azurespappid .Values.acsenginecluster.azurespsecret .Values.acsenginecluster.azuresptenantid .Values.acsenginecluster.kubeconfigprivatekey .Values.acsenginecluster.clientprivatekey -}}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
```

Comment:
You can replace the manually defined labels with your fancy new `labels.standard` helper function. Example here: https://github.com/kubernetes/charts/blob/master/incubator/istio/templates/addons-grafana-deployment.yaml#L8